cmd/terramate/e2etests/create_test.go

```diff
@@ -61,8 +69,12 @@ func TestCreateStack(t *testing.T) {
 			"--import", stackImport2,
 		)
 
+		t.Logf("run create stack %s", stackPath)
```

Comment:
This helped me debug some issues with duplicated IDs on the test...thinking if maybe we should move some t.Logf inside cli.run, so everytime a test involving cli fails we get a full log of all commands and their output etc... WDYT ?

Reply:
hmm not sure because this could generate a lot of noise for cases where the got vs want diff is enough...
let's discuss this in other channels and improve separate if needed.